### ISSUE TITLE
Can not create a CellDataSet object

### ISSUE THREAD

**Issue by Jingle0814** [cole-trapnell-lab/monocle-release#483]
![image](https://user-images.githubusercontent.com/111042514/200515957-75769f96-6cca-452e-89eb-1c56428a4e8d.png)
Hi,
    I have a dataset with 90k cells and I'm failed to create a CellDataSet using monocle2.
    The error message is :

1: In .sparse2dense(e1) :
  sparse->dense coercion: allocating vector of size 20.5 GiB
2: In .sparse2dense(e2) :
  sparse->dense coercion: allocating vector of size 20.5 GiB
Error in if (cds@expressionFamily@vfamily %in% c("negbinomial", "negbinomial.size")) { :
  the condition has length > 1
Calls: reduceDimension -> normalize_expr_data

**Comment by handesome** [comment]
have you figured out this?

**Comment by Jingle0814** [comment]
Have not solved.

**Comment by handesome** [comment]
that's so bad

**Comment by Jingle0814** [comment]
Yes, it's so bad. Because there was no solution, I gave up this method at that time. Do you have any good suggestions?

**Comment by handesome** [comment]
sorry,i don't have a clue.
